### ISSUE TITLE
Support for old format

### ISSUE THREAD

**Issue by zang** [imsky/holder#181]
In v2.9, Is there a way to support the old setup format instead of the new query string?
e.g. holder.js/32x32/text:Sample/#000:#fff

**Comment by imsky** [comment]
not currently. is there an issue with using the query string format?

**Comment by zang** [comment]
not really. There are too many files to change in my app to upgrade to the new format, just wondering if there's a shortcut.

**Comment by imsky** [comment]
not right now, but i think 2.8 or 2.7 supported both formats. let me know if neither of those work for you

**Comment by zang** [comment]
Will try 2.8. 

Actually, I need the support for old formats as the image setup strings are stored in our mongodb. :)

**Comment by zang** [comment]
2.7 works and 2.8 doesn't. It would be great if the old format can be supported in the future release. 
thanks for your help.

**Comment by imsky** [comment]
ok i'll see if i can ship this as an add-on
